packages/server/src/modules/club/club.resolvers.ts

```diff
@@ -15,7 +15,7 @@ export class ClubResolver {
   @Mutation(() => Club)
   public async addClub(
     @Ctx() context: SessionContext,
-    @Arg('name', { nullable: true }) name: string = 'Kool Kidz Klub',
+    @Arg('name') name: string = 'Kool Kidz Klub',
```

Comment:
This should also not be defaulted, it's required by the server so it will always be overwritten